prerelease/action.yaml

```diff
@@ -0,0 +1,190 @@
+name: GitHub Action Release and Publish Prerelease Module
+description: <
+  GitHub Action to compute a prerelease version based on the latest release version and the number of commits since the
+  latest release.
+inputs:
+  checkout-repo:
+    required: false
+    description: Perform checkout as first step of action
+    default: "true"
+  create-prerelease:
+    required: false
+    description: Whether semantic-release should create a prerelease or do a dry run. This can be useful to set to true when a prerelease requires pushing artifacts semantic-release is in charge of generating
+    default: "true"
+  github-token:
+    required: true
+    description: GitHub token that can checkout the repository as well as create tags/releases against it. e.g. 'secrets.GITHUB_TOKEN'
+  extra-plugins:
+    required: false
+    description: Extra plugins for pre-install. You can also specify specifying version range for the extra plugins if you prefer.  Defaults to install @open-turo/semantic-release-config.
+    default: |
+      @open-turo/semantic-release-config
+outputs:
+  new-release-published:
+    description: Whether a new release was published
+    value: ${{ steps.prerelease.outputs.new-release-published }}
+  new-release-version:
+    description: Version of the new release
+    value: ${{ steps.prerelease.outputs.new-release-version }}
+  new-release-major-version:
+    description: Major version of the new release
+    value: ${{ steps.prerelease.outputs.new-release-major_version }}
+  pull-request-number:
+    description: Pull request number
+    value: ${{ steps.PR.outputs.number }}
+  run-url:
+    description: URL to the GHA run
+    value: ${{ steps.vars.outputs.run-url }}
+
+runs:
+  using: composite
+  steps:
+    - name: Dump GitHub context
```

Comment:
needed?

Reply:
It's just really handy to have this when you're trying to see how various github actions are/aren't running, I don't think there's really a penalty to having it here.